electron/src/usb-writer.ts

```diff
@@ -0,0 +1,39 @@
+import * as net from "net"
+import {Device} from "./trusted-main-electron-process/SeedableHardwareKeys/UsbDeviceMonitor";
+import {monitorForFidoDevicesConnectedViaUsb} from "./trusted-main-electron-process/SeedableHardwareKeys/SeedableHardwareKeys";
+import {UsbSocketPath} from "./trusted-main-electron-process/SeedableHardwareKeys/IpcServer";
+import {writeSeedToFIDOKey} from "./trusted-main-electron-process/SeedableHardwareKeys/SeedHardwareKey";
+
+let client = net.createConnection(UsbSocketPath, () => {
+
+    const stopMonitoring = monitorForFidoDevicesConnectedViaUsb((devices: Device[]) => {
+        client.write(JSON.stringify({devices: devices}))
+    }, (error: any) => {
+        client.write(JSON.stringify({error: error}))
+    });
+
+    client.on('data', (data => {
+        let json = JSON.parse(data.toString())
+        if(json.command == 'writeSeedToFIDOKey'){
```

Comment:
Please put interface strings into a file shared by the sender/receiver (client/server)

Reply:
Fixed